plotly/offline/offline.py

```diff
@@ -236,13 +280,32 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
 
     display(HTML(plot_html))
 
+    if image:
+        if image not in __IMAGE_FORMATS:
+            raise ValueError('The image parameter takes only the '
+                             'following format types: `png`, `jpeg`, '
+                             '`webp`, `svg`')
+        # if the check passes then download script injection will commence.
+        # Write script to download image of the plot
+        script = image_download_script('iplot').format(format=image,
+                                                       width=image_width,
+                                                       height=image_height,
+                                                       filename=filename,
+                                                       plot_id=plotdivid)
+        # allow time for the plot to draw
+        import time
```

Comment:
🐄 This import should be at the top of the module. Let me know if you need a better explanation for why this is the case.